From 96a88c1605114db6e1282af3de4d7c3e1a90b9fe Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Tue, 31 Jul 2012 10:56:51 +0800
Subject: [PATCH 037/602] ENGR00215174 add usb camera config.

add usb camera config and default it to false.
it controls which mediaprofile.xml to be added.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 imx6/BoardConfigCommon.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index 4869fc5..29d0ff2 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -30,6 +30,7 @@ BOARD_MODEM_VENDOR := HUAWEI
 BOARD_MODEM_ID := EM750M
 BOARD_MODEM_HAVE_DATA_DEVICE := true
 BOARD_HAVE_IMX_CAMERA := true
+BOARD_HAVE_USB_CAMERA := false
 
 BUILD_WITHOUT_FSL_DIRECTRENDER := false
 BUILD_WITHOUT_FSL_XEC := true
-- 
1.8.0

